src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java

```diff
@@ -239,6 +233,31 @@ private Field schemaToField(Schema sch) {
     return field;
   }
 
+  private Message constructEnumMessage(
+      String name, String description, List<String> enumVals, List<String> enumDescs) {
+    Message enumMessage = new Message(name, false, true, description);
+
+    String dummyDesc = "A dummy enum value indicating that the enum field is not set.";
+    String dummyFieldName = Name.anyCamel("Undefined", name).toUpperUnderscore();
+
+    Field dummyField =
+        new Field(dummyFieldName, Message.PRIMITIVES.get(""), false, null, dummyDesc);
+    enumMessage.getFields().add(dummyField);
+
+    Iterator<String> valIter = enumVals.iterator();
+    Iterator<String> descIter = enumDescs.iterator();
+    while (valIter.hasNext() && descIter.hasNext()) {
+      Field enumField =
+          new Field(valIter.next(), Message.PRIMITIVES.get(""), false, null, descIter.next());
+      if (dummyField.getName().equals(enumField.getName())) {
```

Comment:
Do any APIs use "unset" or "not specified" or "undeclared"? I worry we may miss some other collisions we should catch. Maybe we should check for zero-value enums in the discovery doc.

OTOH, maybe missing the collisions is not pretty, but also not breaking. WDYT?

Reply:
I chose to make the first element in each enum a dummy one, because there is always a chance for an enum to receive a new value alphabetically first in the list. The core difference between enums and messages consists in a requirements to always have a field with a 0 number for enums (message fields cannot have a 0 number at all). 

Also that 0-numbered enum field is considered default for the fields of the enum type. 

I'm not sure what you mean by zero-value enums? There are definitely enums values which already have "Undefined" or something like that in their names. I think searching for those and choosing them as the zero-numbered ones is a good idea. It is not deterministic though (we are basically guessing it here) and not consistent (not all enums, have something like that). Providing a dummy value for each one looked more consistent to me.

In any case, I think trying to find a default value among existent values instead of providing a dummy one is worth exploring, but lets not include it in the scope of this PR.